Source/Logging/Booting/BootLogMessageWriterCreator.cs

```diff
@@ -47,8 +47,14 @@ public void FlushTo(Func<Type, ILogMessageWriter[]> writersForType)
 
                 for (var i = 0; i < writers.Length; ++i)
                 {
-                    if (message.Exception == null) writers[i].Write(message.LogLevel, message.Message, message.Arguments);
-                    else writers[i].Write(message.LogLevel, message.Exception, message.Message, message.Arguments);
+                    try
+                    {
+                        if (message.Exception == null) writers[i].Write(message.LogLevel, message.Message, message.Arguments);
+                        else writers[i].Write(message.LogLevel, message.Exception, message.Message, message.Arguments);
+                    }
+                    catch
+                    {
+                    }
```

Comment:
The empty `catch` block is pretty scary in a way as we can't ignore errors coming from logging itself, how else do we know we cant or can't trust them? Maybe have a simple log there about "exception while logging" so that we know that we can't take the logs produced from a particular run at face value?

It kinda feels like we'd need some retry things for logging too in case of something like this happens.

Reply:
Yep - I agree, hence my comment in the description that we need to do this in the future. However, logging errors when the logging isn't working is not trivial.

So although I agree that it would be nice to output these kinds of errors somewhere, we need to make a conscious decision about that, and this is not new. The exact same thing can happen in `InternalLogger`.

This PR was not intended to fix disappearing log messages, but rather make the behaviour during boot logging the same as _normal_ logging.

Reply:
If we want to keep track of this issue - I suggest we make an issue on GitHub to fix this behaviour separately.

Reply:
https://github.com/dolittle/Home/issues/84